### PR TITLE
VZ-7427: Build Ingress controller 1.3.1 from source

### PR DESCRIPTION
**Pull request by bhatpmk**
Builds Ingress controller 1.3.1 from source.

### CODE REVIEW

**Review by michael-cico — APPROVED** [review]

File: build-images.sh [hunk 1]
```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# Copyright (c) 2022, Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
```
Comment (guoyongzhang) [inline build-images.sh:4]: remove this line.  Can not add Universal Permissive License to this project which is under Apache 2.0 license.  Actually this line should be removed from all files in this project.

File: images/custom-error-pages/Makefile [hunk 2]
```diff
@@ -12,6 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Copyright (c) 2022, Oracle and/or its affiliates.
```
Comment (guoyongzhang) [inline images/custom-error-pages/Makefile:15]: Remove this line.  Oracle copyright should not be added for existing files.

File: build-images.sh (hunk 1 above)
Comment (bhatpmk) [reply build-images.sh]: Done, thank you @guoyongzhang

File: images/custom-error-pages/Makefile (hunk 2 above)
Comment (bhatpmk) [reply images/custom-error-pages/Makefile]: Updated the files accordingly

**Review by guoyongzhang — APPROVED** [review]
looks good from copyright/license perspective

File: images/nginx/rootfs/Dockerfile [hunk 3]
```diff
@@ -11,62 +11,97 @@
...
+    nettle-3.4.1-7.el8.x86_64 \
+  && rpm -i https://yum.oracle.com/repo/OracleLinux/OL7/latest/x86_64/getPackage/GeoIP-1.5.0-13.el7.x86_64.rpm \
+  && rpm -i https://yum.oracle.com/repo/OracleLinux/OL7/optional/developer/x86_64/getPackage/GeoIP-devel-1.5.0-13.el7.x86_64.rpm \
+  && chmod +x /usr/bin/dumb-init \
   && ln -s /usr/local/nginx/sbin/nginx /sbin/nginx \
+  && ln -s /usr/local/lib/libluajit-5.1.so.2 /lib64/libluajit-5.1.so.2 \
+  && addgroup -Sg 101 www-data \
   && adduser -S -D -H -u 101 -h /usr/local/nginx \
-  -s /sbin/nologin -G www-data -g www-data www-data \
+    -s /sbin/nologin -G www-data -g www-data www-data \
```
[the remark this answers is not in the document]
Comment (bhatpmk) [reply images/nginx/rootfs/Dockerfile:91]: It provides better readability

**Pull request merged**